Revit_Core_Engine/Query/LookupParameter.cs

```diff
@@ -42,7 +42,11 @@ public static partial class Query
         [Input("parameterNames", "Names of the parameter to be iterated over in search for the parameter.")]
         [Input("parameterGroups", "If not null, only the Revit parameters from the given parameter groups will be parsed.")]
         [Output("parameter", "Parameter extracted from the input Revit element.")]
+#if REVIT2020 || REVIT2021 || REVIT2022
```

Comment:
Same as previously - in general, the comment concerns all places where you added an `#if` that spans only between 2020 and 2022 rather than 2024